app/models/user.rb

```diff
@@ -28,7 +28,6 @@ class User < ApplicationRecord
             notify_email: true
 
   validates :uid, uniqueness: { allow_blank: true }
-  validates :uid, inclusion: { in: ->(user) { [user.uid_was] } }, on: :npq_separation, if: -> { uid_was.present? }
   # TODO: add constraints into the DB after separation
```

Comment:
Should we remove those conditions and add DB constraints as well in this another? or Is it in another ticket?

Reply:
The `TODO:`'s are for another ticket 👍